src/commands/feedback.js

```diff
@@ -1,6 +1,57 @@
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
 
+async function collector(interaction, message, embed) {
+
+	const filterReaction = (reaction, user) => {
+		return !user.bot && (reaction.emoji.name === '🟩' || reaction.emoji.name === '🟨' || reaction.emoji.name === '🟥');
+	};
+
+	const collectorReaction = message.createReactionCollector({ filter: filterReaction });
+
+	collectorReaction.on('collect', async (reaction, user) => {
+		if (reaction.emoji.name === '🟩') {
+			await message.reactions.removeAll();
+			try {
+				interaction.options.getSubcommand() === 'sugestão' ? await interaction.user.send('Olá! Obrigado pela sugestão! A equipe de desenvolvedores agradece!') : await interaction.user.send('Olá! Obrigado por reportar o bug! A equipe de desenvolvedores já está ciente do mesmo e logo logo ele estará resolvido!');
+			}
+			catch {
+				await message.channel.send('Não foi possível enviar a mensagem na dm do usuário.');
+			}
+			embed.setColor('GREEN');
+			interaction.options.getSubcommand() === 'sugestão' ? embed.setFooter({ text: `Sugerido por ${interaction.user.username} (${interaction.user.id}) - Aprovado por ${user.username}`, iconURL: interaction.user.avatarURL() }) : embed.setFooter({ text: `Reportado por ${interaction.user.username} (${interaction.user.id}) - Aprovado por ${user.username}`, iconURL: interaction.user.avatarURL() });
+			await message.edit({ embeds: [embed] });
+			message.pin();
+		}
+		else if (reaction.emoji.name === '🟨') {
+			await message.reactions.removeAll();
+			const messageAnswer = await message.channel.send('Escreva a resposta:');
+
+			const filter = (msg) => user.id === msg.author.id;
+			const sendedMessage = await message.channel.awaitMessages({ max: 1, filter }).then(async (msg) => {
```

Comment:
```suggestion
			const sendedMessage = await message.channel.awaitMessages({ max: 1, filter }).then(msg => {
```
This function doesn't have any `await` task, so the `async` keyword isn't necessary.